xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-JSONRPC_VERSION 10.5.2
+JSONRPC_VERSION 10.5.3
```

Comment:
Adding a property at the end is a non-breaking change to the API, hence bump the minor (middle) number 10.6.0
See https://github.com/xbmc/xbmc/blob/88fffa1392bd59914a4050d0a52ab6799c11a54b/xbmc/interfaces/json-rpc/schema/methods.json#L33

Reply:
Changed.